warehouse/query-core/src/main/java/datawave/query/jexl/lookups/LookupTermsFromRegex.java

```diff
@@ -159,8 +141,8 @@ public IndexLookupMap lookup(ShardQueryConfiguration config, ScannerFactory scan
                 
             }
             
-            for (Text field : fields) {
-                opts.fetchColumnFamily(field);
+            for (String field : fields) {
```

Comment:
This seems to assume that fields will always be a superset of reverseFields. I think we might need to intersect the two sets to produce correcte behavior here.